### ISSUE TITLE
Screenreaders announce input fields as `stepper` or `telephone` when restricting inputType to numbers

### ISSUE THREAD

**Issue by afaiz-demyst** [devfolioco/react-otp-input#463]
- **Describe the Feature Request**
  I have the following requirements
  - allow only numeric inputs to the fields
  - when using a screenreader, do not announce the field as stepper, or as a telephone input field.
    - setting `inputType` as `number` or `tel` will solve the first issue, but violates the 2nd

- **Describe Preferred Solution**
  somehow expose `isInputNum` again.
